src/Compilers/CSharp/Test/Symbol/Symbols/TypeTests.cs

```diff
@@ -37,6 +35,9 @@ public void Bug18280()
             var arr = x.Type.TypeSymbol;
 
             arr.GetHashCode();
+            // PROTOTYPE(NullableReferenceTypes): StackOverflowException in
```

Comment:
>// PROTOTYPE(NullableReferenceTypes): StackOverflowException in [](start = 12, length = 63)

Why leave this comment if test passes?

Reply:
The commented out line will result in a `StackOverflowException` when uncommented.

---
In reply to: [214486835](https://github.com/dotnet/roslyn/pull/29607#discussion_r214486835) [](ancestors = 214486835)